Source/Scene/ModelExperimental/SceneMode2DPipelineStage.js

```diff
@@ -70,12 +74,16 @@ SceneMode2DPipelineStage.process = function (
     frameState
   );
 
+  const runtimePrimitive = renderResources.runtimePrimitive;
   runtimePrimitive.boundingSphere2D = boundingSphere2D;
 
-  const positionAttribute = ModelExperimentalUtility.getAttributeBySemantic(
-    primitive,
-    VertexAttributeSemantic.POSITION
-  );
+  // If the model is instanced, 2D projection will be handled in the
+  // InstancingPipelineStage. Unload the typed array and return early.
+  const instanced = defined(renderResources.runtimeNode.node.instances);
+  if (instanced) {
+    positionAttribute.typedArray = undefined;
```

Comment:
Hm... is it possible to check for this situation at loading time so we don't load the typed array in the first place?